cmd/internal/shared/devicerepository/config.go

```diff
@@ -14,12 +14,21 @@
 
 package devicerepository
 
-import "go.thethings.network/lorawan-stack/v3/pkg/devicerepository"
+import (
+	"go.thethings.network/lorawan-stack/v3/pkg/devicerepository"
+	"go.thethings.network/lorawan-stack/v3/pkg/devicerepository/store/bleve"
+)
 
 // DefaultDeviceRepositoryConfig is the default configuration for the Device Repository.
 var DefaultDeviceRepositoryConfig = devicerepository.Config{
-	ConfigSource: "url",
-	URL:          "https://raw.githubusercontent.com/TheThingsNetwork/lorawan-devices/master",
+	ConfigSource: "directory",
+	Directory:    "data/lorawan-devices",
+
+	Store: devicerepository.StoreConfig{
+		Bleve: bleve.Config{
+			SearchPaths: []string{"device-repository", "/srv/ttn-lorawan/device-repository"},
```

Comment:
Same story as in my Dockerfile comment: This dirname may be a bit confusing, maybe we should make clear that it contains the index, not the entire repository.